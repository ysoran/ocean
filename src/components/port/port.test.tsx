import React from 'react';
import { shallow } from 'enzyme';
import Port from './port';

const setup = () => {
    const wrapper = shallow(<Port lat={0} lng={0} />);
    return {
        wrapper,
    };
};

describe('Port component', () => {
    it('should render snapshot', () => {
        const { wrapper } = setup();
        expect(wrapper).toMatchSnapshot();
    });
});
