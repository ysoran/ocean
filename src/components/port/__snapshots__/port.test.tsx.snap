// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Port component should render snapshot 1`] = `ShallowWrapper {}`;
